freyja/1.3/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM continuumio/miniconda:4.7.12
+
+# Version arguments
+ARG FREYJA_SOFTWARE_VERSION="1.3"
+
+LABEL base.image="continuumio/miniconda:4.7.12"
+LABEL dockerfile.version="1"
+LABEL software="Freyja"
+LABEL software.version=${FREYJA_SOFTWARE_VERSION}
+LABEL description="Freyja is a tool to recover relative lineage abundances from mixed SARS-CoV-2 samples from a sequencing dataset (BAM aligned to the Hu-1 reference)"
+LABEL website="https://github.com/andersen-lab/Freyja"
+LABEL license="https://github.com/andersen-lab/Freyja/blob/main/LICENSE"
+LABEL maintainer="Kevin Libuit"
+LABEL maintainer.email="kevin.libuit@theiagen.com"
+
+# Create Freyja conda environment and add to PATH
+RUN conda create -n freyja-env -c defaults -c bioconda -c conda-forge freyja=${FREYJA_SOFTWARE_VERSION} \
```

Comment:
It's strongly recommended by bioconda to set the channels in this specific order:
conda-forge - top priority
bioconda
defaults - lowest priority

The way you have it set right now is in opposite order.

I recommend changing this line to:
```
conda create -n freyja-env -c conda-forge -c bioconda -c defaults freyja=${FREYJA_SOFTWARE_VERSION}
```

I'm getting this message while building and it's taking a looong time (>5min) to solve the environment
```
Step 12/14 : RUN conda create -n freyja-env -c defaults -c bioconda -c conda-forge freyja=${FREYJA_SOFTWARE_VERSION} && echo "source activate freyja-env" > ~/.bashrc
 ---> Running in 6228ea900e9d
Collecting package metadata (current_repodata.json): ...working... done
Solving environment: ...working... failed with repodata from current_repodata.json, will retry with next repodata source.
Collecting package metadata (repodata.json): ...working... done
Solving environment: ...working...

```

Reply:
oops, I meant to comment ^ on the 1.3.1 dockerfile, not 1.3. The same still applies though